frontend/src/components/grantaccess/GrantAccess.vue

```diff
@@ -189,19 +181,30 @@ async function verifyForestClientNumber(forestClientNumber: string) {
     }
 
     //The input is updated with only the numbers that have errored out. The array is converted to a string values comma separated
-    formData.value['forestClientNumber'] = forestNumbers.toString();
-}
+    formData.value['forestClientNumbers'] = forestNumbers.toString();
+};
 
-function isForestClientNumberNotAdded(forestClientNumber: string) {
+const ifForestClientNumberNotAdded = (forestClientNumber: string) => {
```

Comment:
rename this method from "isForestClientNumberNotAdded" to "ifForestClientNumberNotAdded", so the logic when call this method is easier to understand

Reply:
The best practice for naming **boolean** variables (or methods) is using "**is**" "**has**" "**should**" "**can**" with **positive** wording. I would strongly favor to use them, although sometimes it is hard to find a good naming. 
For example, `if ( `**if**ForestClientNumberNotAdded()` )` would be really strange when other developers see that.

Reply:
sure I'll revert it back, sometimes it's hard to give a name :   )